Cargo.toml

```diff
@@ -30,7 +31,6 @@ iroha_data_model_derive = { version = "=2.0.0-pre-rc.20", path = "data_model/der
 iroha_client = { version = "=2.0.0-pre-rc.20", path = "client" }
 iroha_config = { version = "=2.0.0-pre-rc.20", path = "config" }
 iroha_config_base = { version = "=2.0.0-pre-rc.20", path = "config/base" }
```

Comment:
move to `config/Cargo.toml`, it's not used elsewhere

Reply:
It is also used in `iroha_client`.

In future `iroha_config_base` will be replaced with an external generic config library.